http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstall.scala

```diff
@@ -130,12 +131,15 @@ class CromwellRunnerAppInstall[F[_]](config: CromwellRunnerAppConfig,
   override def checkStatus(baseUri: Uri, authHeader: Authorization)(implicit ev: Ask[F, AppContext]): F[Boolean] =
     cromwellDao.getStatus(baseUri, authHeader).handleError(_ => false)
 
-  def toCromwellRunnerAppDatabaseNames(dbNames: List[String]): Option[CromwellRunnerAppDatabaseNames] =
-    (dbNames.find(name => name.startsWith("cromwell") && !name.startsWith("cromwellmetadata")),
-     dbNames.find(_.startsWith("tes")),
-     dbNames.find(_.startsWith("cromwellmetadata"))
-    )
-      .mapN(CromwellRunnerAppDatabaseNames)
+  def toCromwellRunnerAppDatabaseNames(
+    dbResources: List[WsmControlledDatabaseResource]
+  ): Option[CromwellRunnerAppDatabaseNames] =
+    (dbResources
+       .find(db => db.wsmDatabaseName.startsWith("cromwell") && !db.wsmDatabaseName.startsWith("cromwellmetadata"))
```

Comment:
since this logic depends on the db name starting with the app type, is it possible to rename the `cromwellmetadata` db to something that won't cause issue since we also have a `cromwell` db type?

Reply:
The database names here are assumed to be prefixed with the prefix mentioned in the databases list in each AppInstall (for example [this](https://github.com/DataBiosphere/leonardo/blob/9354890a356334202adff9d83074811f40ee328a/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstall.scala#L31-L36)) instead of the app type. I think we can rename the `cromwellmetadata` database to something else but that would be a bigger lift and out of context of this PR.

Reply:
that works for me, it feels a little brittle at the moment.

Reply:
Yeah I agree that this check does feel little brittle but it will be hard to change the names of database as it will require helm changes too. In case it helps, the main logic for the condition that is being checked here is same as before but only uses the WSM database name instead of directly Azure name.